resources/views/components/button/index.blade.php

```diff
@@ -1,3 +1,3 @@
-<x-rapidez::button.base {{ $attributes->class('border') }}>
+<x-rapidez::button.base {{ $attributes->class('bg-emphasis border') }}>
```

Comment:
We don't need a hover here? But maybe something for https://github.com/rapidez/blade-components/pull/8

Reply:
The buttons will be replaced with the buttons from the blade-components. So in there buttons will have a hover state.